docs/pages/identity-governance/device-trust/intune-integration.mdx

```diff
@@ -0,0 +1,172 @@
+
+[Grant the application permission](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-configure-app-access-web-apis#application-permission-to-microsoft-graph)
+to read Intune managed devices:
+
+1. In your registered application, navigate to API permissions > Add a permission.
+2. Select Microsoft Graph > Application permissions.
+3. Search for and add the `DeviceManagementManagedDevices.Read.All` permission.
+4. Click "Grant admin consent" for your organization and confirm.
+
+<Admonition type="note">
+The consent must be granted by a Global Administrator or Privileged Role Administrator. Failure to
+grant this permission will result in 401 Forbidden errors from the Graph API.
+</Admonition>
+
+To [create a client secret](https://learn.microsoft.com/en-us/graph/auth-register-app-v2#option-2-add-a-client-secret):
+
+1. In your registered application, navigate to Certificates & secrets > Client secrets.
+2. Click New client secret.
+3. Add a description and select an expiration period (we recommend setting a reminder to rotate
+   the secret before it expires).
+4. Click "Add" and copy the Value (not the Secret ID).
+
+## Step 3/3. Configure the Intune integration in Teleport
+
+1. In the Web UI, navigate to Add New > Integration and select "Microsoft Intune".
+2. Fill in the required information:
+   - Primary domain or Microsoft Entra tenant ID: Enter your [tenant identifier](https://learn.microsoft.com/en-us/partner-center/account-settings/find-ids-and-domain-names#find-the-microsoft-entra-tenant-id-and-primary-domain-name)
+     (e.g., `contoso.onmicrosoft.com` or the tenant GUID).
+   - Application (client) ID: Paste the Application ID you copied in Step 1.
+   - Client secret value: Paste the client secret value you copied in Step 2.
+3. Click "Connect Microsoft Intune" to complete the setup.
+
+The integration will begin syncing devices from your Intune inventory within two minutes. Depending
+on the size of your inventory, the initial sync may take a few minutes to complete.
+
+## How syncing works
+
+### Sync schedule
+
+The Intune integration uses the following fixed schedule that starts when the integration is
+installed, updated, or when the Auth Service is restarted:
+
+- Full sync: every 24 hours – fetches all devices from Intune, reconciles the complete
+  inventory and removes missing devices.
+- Partial sync: every 6 hours between full syncs – fetches only devices that have been modified in
+  Intune since the last sync.
+
+The integration tracks the last sync time from Intune to fetch only changed devices during partial
+syncs.
+
+### Device requirements
+
+For a device to be synced, it must have a serial number, its device registration state must be set
+to "registered" and its operating system must be either macOS, Windows, or Linux.
+
+### Device ownership and removal
+
+When syncing inventory, the Intune integration claims ownership of all synced devices.
```

Comment:
Does _claims ownership_ mean that if the device already exists (via manual registration) it will start being managed by the Intune integration?